package.json

```diff
@@ -12,8 +12,8 @@
     "lint": "eslint . --ext .js,.html",
     "postinstall": "npm run build:icons",
     "test": "npm run lint && npm run test:local && npm run test:diff",
-    "test:diff": "mocha './**/*.visual-diff.js' -t 10000",
-    "test:diff:golden": "mocha './**/*.visual-diff.js' -t 10000 --golden",
+    "test:diff": "mocha ./**/*.visual-diff.js -t 10000",
```

Comment:
Were the single quotes causing problems?

Reply:
yes, for some reason it was unable to locate my .js files in local test. Margaree suggested the fix, and the test run fine now.